packages/components/src/VirtualizedList/ListTable/ListTable.component.js

```diff
@@ -50,8 +52,8 @@ function ListTable(props) {
 			rowClassName={({ index }) =>
 				classNames(...['tc-list-item', rowThemes, collection[index] && collection[index].className])
 			}
-			rowCount={collection.length}
-			rowGetter={({ index }) => collection[index]}
+			rowCount={totalRowCount || collection.length}
```

Comment:
`collection.length` may not reflect the actual amount of rows in the collection (if not fully loaded yet).